modepy/tools.py

```diff
@@ -101,6 +101,13 @@ def simplex_integral(self):
                 * reduce(mul, (factorial(alpha) for alpha in self.exponents))
                 / factorial(len(self.exponents)+sum(self.exponents)))
 
+    def hypercube_integral(self):
+        """Integral over the unit hypercube."""
```

Comment:
```suggestion
        """Integral over :math:`[0,1]^n`."""
```

Reply:
If you're feeling kind, maybe also define the "unit simplex" above.

Reply:
Fixed in a4363b0.

Is there a more compact expression for the simplex?

Reply:
Thx! Not that I know of.